src/main.ts

```diff
@@ -52,6 +52,14 @@ async function start () {
 
   setBaseRouteAndBusinessId(windowLocationPathname, processEnvBaseUrl, windowLocationOrigin) // may throw an error
 
+  // Saftey check for proper business number
+  const businessId = sessionStorage.getItem('BUSINESS_ID')
+  console.log(`Business ID: ${businessId}`)
+  if (!businessId) {
+    const businessRegistryDashboardUrl = configurationStore.getBusinessRegistryDashboardUrl
+    navigate(businessRegistryDashboardUrl)
+    return
+  }
```

Comment:
1. Would it make sense to move this into setBaseRouteAndBusinessId(), or lines 137+ below?
2. What about checking for valid T number?
3. Previously, setBaseRouteAndBusinessId threw an error, which was caught and handled on lines 137+ below. Is that still useful?
4. Is the console log for debugging? Please remove it.

Reply:
I'm OK with keeping the code here.

What do you think about throwing an error here and letting the catch block on lines 136+ handle the navigation?

You need the import to call the store getter, right? What if you only check for a BUSINESS_ID and then throw an error? Now your code could go back into set-base-route-and-business-id...?